### PR TITLE
SOV-2805: Staking and Rewards page UX updates

### PR DESCRIPTION
**Pull request by Rickk137**
https://sovryn.atlassian.net/browse/SOV-2805

### CODE REVIEW

File: apps/frontend/src/app/3_organisms/StakeForm/components/AddStakeForm/AddStakeForm.tsx [hunk 1]
```diff
@@ -232,7 +236,7 @@ export const AddStakeForm: FC<StakeFormProps> = ({
           />
         )}
         <SimpleTableRow
-          label={t(translations.stakePage.stakeForm.votingPowerReceived)}
+          label={t(translations.stakePage.stakeForm.votingPowerIncrease)}
```
Comment (tiltom) [inline apps/frontend/src/app/3_organisms/StakeForm/components/AddStakeForm/AddStakeForm.tsx:239]: Delete `stakeForm.votingPowerReceived` from `translations` as it's not used anywhere.

File: apps/frontend/src/app/5_pages/RewardsPage/RewardsPage.tsx [hunk 2]
```diff
@@ -89,9 +93,59 @@ const RewardsPage: FC = () => {
         <title>{t(translations.rewardPage.meta.title)}</title>
       </Helmet>
 
-      <div className="flex flex-col lg:flex-row gap-2 lg:gap-20 w-full lg:w-5/6 text-gray-10 mt-6 lg:mt-12">
-        <div className="w-full lg:w-80">
+      <div className="flex flex-col lg:flex-row gap-2 lg:gap-16 w-full lg:w-5/6 text-gray-10 mt-6 lg:mt-12">
+        <div className="flex flex-col gap-4 w-full lg:w-96">
           <TotalRewardsEarned />
+
+          <Banner
```
Comment (tiltom) [inline apps/frontend/src/app/5_pages/RewardsPage/RewardsPage.tsx:103]: The banners should have `Learn more` links.

Design:
image

Implementation:
image

**Review by tiltom — CHANGES_REQUESTED** [review]
There's no top-banner on the Staking page.

Design:
image

Implementation:
image

**Comment by Rickk137** [comment]
> There's no top-banner on the Staking page.
> 
> Design: image
> 
> Implementation: image

The banner can be closed on mobile. if you do so you won't see it any more.

File: apps/frontend/src/locales/en/translations.json [hunk 3]
```diff
@@ -553,6 +553,21 @@
...
             }
         },
+        "stabilityPoolBanner": {
+            "title": "Stability Pool",
+            "description": "Earn from Zero loans by providing DLLR, 5%-110% APY",
+            "action": "Deposit DLLR"
+        },
+        "stakingBanner": {
+            "title": "Staking",
+            "description": "Participate in Sovryn's governance and revenue, earn up to 26.8% APY paid in bitcoin",
```
Comment (tiltom) [inline apps/frontend/src/locales/en/translations.json]: Verify with @john-light if this number should be fetched dynamically or if a static value is enough.

**Review by tiltom — APPROVED** [review]

File: apps/frontend/src/locales/en/translations.json (hunk 3 above)
Comment (john-light) [reply apps/frontend/src/locales/en/translations.json]: It should be dynamic, based on the same method used to get the staking APY on the staking page.

**Pull request closed without merging**